GitUI/SplitterManager.cs

```diff
@@ -0,0 +1,192 @@
+﻿using GitUIPluginInterfaces;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace GitUI
+{
+    public class SplitterManager
+    {
+        private readonly ISettingsSource _settings;
+        private readonly List<SplitterData> splitters = new List<SplitterData>();
+        private float _designTimeFontSize;
+
+        public SplitterManager(ISettingsSource settings, float designTimeFontSize = 8.25F)
```

Comment:
Such a scaling is aplied by GitExtensionsForm.RestorePosition. This changes the form's dimensions.
Then I scale the splitterDistance in proportion to the new form's dimensions to the design time form's dimensions. I am unbale to test it on high DPI srceen, because I don't have a one. It works well when the app font size is changed and when you connect via remote desktop with different screen dimensions.